### DISCUSSION TITLE
Hear a particular phrase

### DISCUSSION THREAD

**Discussion by Lex-Kent** [speechly/speechly#300] [Q&A]
Hi all,

I'm struggling with a particular phrase in my app. My client has an acronym which is "OFA". The user of the app might use this phrase is a number of places. The one I'm really struggling with is an open help area in which my bot asks the user "What can I help you with?". Of course, this allows for any kind of response which makes creating examples difficult (I _have_ created an application which attempts to show some example questions, but it doesn't seem to be helping at all).

Today I repeated the question "What does OFA stand for?" maybe 20 times in a row and no matter how clearly I spoke, Speechly never transcribed it correctly.

How can I create an application that will steer the transcription in the direction of common phrases that might appear anywhere in a question or response please?

Thanks so much for any help!

**Comment by teelisyys** [comment]
Hi @Lex-Kent ! Thank you for using Speechly!

Please be aware that currently the expected output for this kind of acronym is in spelled out form `o f a`.  The orthographic form `OFA` will not be produced.

Also note that any _Training data_ / _Text adaptation_ should use the spelled out form to be effective. A quick test suggest that at least in a simple case with text adaptation, the `o f a` is being recognised. 

If the text adaptation does not solve the issue, our [audio adaptation](https://docs.speechly.com/features/model-adaptation#audio-adaptation) feature, available on the [Enterprise tier](https://www.speechly.com/pricing), is well suited to teach our speech recognition model your domain specific vocabulary, along with tuning the model to the acoustic environment you're app is used in.

**Reply by Lex-Kent** [reply]
Hi @teelisyys , thanks so much for responding so quickly. I've taken your advice and added a bunch of example phrases containing the acronym spelled out. I am now being heard correctly more often than before, but I'm still misheard more than half the time.

Unfortunately my client is still not ready to move up to the enterprise tier, so I guess I'll just have to live with this for now :o(

thanks again,

**Reply by ghost** [reply]
@Lex-Kent I am wondering if you have tired evaluating the ASR using Speechly via command line tool. https://docs.speechly.com/features/evaluate-asr
By recording the voice and running the test locally.

Not sure if this will help, I haven't tried yet. Also, Not entirely sure if this type of test requires the enterprise plan. Not mentioned on the provided link.
